MagicianHWInterface:
  magician_arm_controller:
    type: position_controllers/JointGroupPositionController
    joints:
       - magician_joint1
       - magician_joint2
       - magician_joint3
    state_publish_rate:  25


  joint_state_controller:
      type: joint_state_controller/JointStateController
      publish_rate: 50 
      

